systems/overview/12-Glossary.md

```diff
@@ -26,14 +26,34 @@ A component is a software unit with a well-defined interface and explicitly spec
 Extensions are software components that administrators can add (or remove) to their MediaWiki application in order to customize how it looks and works.
 Examples: WikibaseClient, UniversalLanguageSelector, OAuth
 
+## [WikibaseClient](https://www.mediawiki.org/wiki/Extension:Wikibase_Client)
+
+WikibaseClient is a [MediaWiki Extension](#mediawiki-extension) that enables the use and display of Entity data from a [Wikibase Repository](#wikibase-repository) via Lua modules or parser functions. Clients can also use centralized language links.
+
+## Wikibase Repository
```

Comment:
Maybe we can also link to the extension page in the title.

Reply:
I deliberately linked to the extension page only from the description but not from the title to make it clearer that we're talking about [the Repo application, not the extension](https://github.com/wmde/wikidata-wikibase-architecture/blob/main/docs/decisions/client-repo-mediawiki.md) here. Not sure if that needs to be made clearer here or if it's fine to link to it here anyway.

Reply:
Oh ok, then maybe we can add a sentence to the description saying "It can optionally contain WikibaseLexeme etc." ?

Reply:
Yup that makes sense. Added.